packages/frontend/src/redux/slices/transactions/index.js

```diff
@@ -88,26 +89,10 @@ const transactionsSlice = createSlice({
         }
     },
     extraReducers: ((builder) => {
-        builder.addCase(fetchTransactions.pending, (state, { meta }) => {
-            const { accountId } = meta.arg;
-
-            set(state, ['byAccountId', accountId, 'status', 'loading'], true);
-            set(state, ['byAccountId', accountId, 'status', 'error'], initialErrorState);
-        });
-        builder.addCase(fetchTransactions.fulfilled, (state,  { meta }) => {
-            const { accountId } = meta.arg;
-
-            set(state, ['byAccountId', accountId, 'status', 'loading'], false);
-            set(state, ['byAccountId', accountId, 'status', 'error'], initialErrorState);
-        });
-        builder.addCase(fetchTransactions.rejected, (state, { meta, error }) => {
-            const { accountId } = meta.arg;
-            
-            set(state, ['byAccountId', accountId, 'status', 'loading'], false);
-            set(state, ['byAccountId', accountId, 'status', 'error'], {
-                message: error?.message || 'An error was encountered.',
-                code: error?.code
-            });
+        handleAsyncThunkStatus({
+            asyncThunk: `${SLICE_NAME}/fetchTransactions`,
```

Comment:
Per feedback above would prefer to use `fetchTransactions.typePrefix` here instead.